apps/cugetreg-api/src/review/review.service.ts

```diff
@@ -32,6 +32,13 @@ export class ReviewService {
     { courseNo, semester, academicYear, studyProgram, rating, content }: CreateReviewInput,
     userId: string
   ): Promise<Review> {
+    if (rating) {
```

Comment:
I don't get this line. How does the bound check work?

Reply:
I derped. This is supposed to be 'if (rating < 0 || rating > 10)'

Same as below